src/ocean/net/email/EmailSender.d

```diff
@@ -162,7 +162,8 @@ class EmailSender
         {
             try
             {
-                execute;
+                setArgs("sendmail", "-t", "-f", sender).execute();
```

Comment:
I'm not sure if From should always be the same as Sender address. Also, perhaps less intrusive way would be to make setting sender optional?

Reply:
Good point. But I think that would be a new feature, this bug-fix is mainly to prevent the envelope sender is empty